lib/sie/parser.rb

```diff
@@ -33,8 +37,14 @@ def parse(io)
 
     private
 
+    attr_reader :options
```

Comment:
We have https://github.com/barsoom/attr_extras as a dependency for this library so you can do "attr_private :options" up beside initialize.